tests/unit/dojson/test_dojson_hep_bd2xx.py

```diff
@@ -0,0 +1,138 @@
+        '</datafield>'
+    )  # record/001511698
+
+    expected = [
+        {
+            'title': 'Exact Form of Boundary Operators Dual to Interacting '
+                     'Bulk Scalar Fields in the AdS/CFT Correspondence',
+            'source': 'arXiv',
+        },
+    ]
+    result = hep.do(create_record(snippet))
+
+    assert validate(result['titles'], subschema) is None
+    assert expected == result['titles']
+
+    expected = [
+        {
+            'a': 'Exact Form of Boundary Operators Dual to Interacting '
+                 'Bulk Scalar Fields in the AdS/CFT Correspondence',
+            '9': 'arXiv',
+        },
+    ]
+    result = hep2marc.do(result)
+
+    assert expected == result['245']
+
+
+def test_titles_from_245_and_246__a_9():
+    schema = load_schema('hep')
+    subschema = schema['properties']['titles']
+
+    snippet = (
+        '<datafield tag="245" ind1=" " ind2=" ">'
+        '  <subfield code="a">Superintegrable relativistic systems in'
+        ' spacetime-dependent background fields</subfield>'
+        '  <subfield code="9">arXiv</subfield>'
+        '</datafield>'
+        '<datafield tag="246" ind1=" " ind2=" ">'
+        '  <subfield code="a">Superintegrable relativistic systems in'
+        ' spacetime-dependent background fields</subfield>'
+        '  <subfield code="9">arXiv</subfield>'
+        '</datafield>'
+    )  # record/1511471
+
+    expected = [
+        {
+            'title': 'Superintegrable relativistic systems in '
+                     'spacetime-dependent background fields',
+            'source': 'arXiv',
+        },
+    ]
+    result = hep.do(create_record(snippet))
+
+    assert validate(result['titles'], subschema) is None
+    assert expected == result['titles']
+
+    expected = [
+        {
+            'a': 'Superintegrable relativistic systems in spacetime-dependent background fields',
+            '9': 'arXiv',
+        },
+    ]
+    result = hep2marc.do(result)
+
+    assert expected == result['245']
+
+
+def test_titles_from_245__a_b():
+    schema = load_schema('hep')
+    subschema = schema['properties']['titles']
+
+    snippet = (
+        '<datafield tag="245" ind1=" " ind2=" ">'
+        '  <subfield code="a">Proceedings, New Observables in Quarkonium Production</subfield>'
+        '  <subfield code="b">Trento, Italy</subfield>'
+        '</datafield>'
+    )  # record/1510141
+
+    expected = [
+        {
+            'title': 'Proceedings, New Observables in Quarkonium Production',
+            'subtitle': 'Trento, Italy',
+        },
+    ]
+    result = hep.do(create_record(snippet))
+
+    assert validate(result['titles'], subschema) is None
+    assert expected == result['titles']
+
+    expected = [
+        {
+            'a': 'Proceedings, New Observables in Quarkonium Production',
+            'b': 'Trento, Italy',
+        },
+    ]
+    result = hep2marc.do(result)
+
+    assert expected == result['245']
```

Comment:
You should add a test for the case where the two titles in `245` and `246` are different, e.g. http://inspirehep.net/record/36035. In that case, the `245` goes to the first element of `titles`, the `246`s go next.